apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/steps/FinishedStep.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+
+import ICON_SOCIAL_X from '@cowprotocol/assets/images/icon-social-x.svg'
+import LOTTIE_GREEN_CHECKMARK_DARK from '@cowprotocol/assets/lottie/green-checkmark-dark.json'
+import LOTTIE_GREEN_CHECKMARK from '@cowprotocol/assets/lottie/green-checkmark.json'
+import { ExplorerDataType, getExplorerLink, getRandomInt, isSellOrder, shortenAddress } from '@cowprotocol/common-utils'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { TokenLogo } from '@cowprotocol/tokens'
+import { Confetti, ExternalLink, InfoTooltip, TokenAmount } from '@cowprotocol/ui'
+import { CurrencyAmount } from '@uniswap/sdk-core'
+
+import Lottie from 'lottie-react'
```

Comment:
Out of this task scope.
This `lottie-react` lib is pretty heavy, would be great to lazy-load this `FinishedStep` component in order to load the library only we display the component.